### PR TITLE
document HTML Format and Run Query - Advanced nodes

### PR DESCRIPTION
**Pull request by john-floren-gravwell**
This PR addresses #580

### CODE REVIEW

File: flows/nodes/runqueryadvanced.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Run Query - Advanced Node
+
+This node executes a Gravwell query, much like the [Run a Query](runquery) node, but provides more advanced options for specifying the search duration. It outputs a structure into the payload (named `search` by default) which contains information about the search and allows other nodes to access the results.
+
+## Configuration
+
+* `Query String`, required: the Gravwell query to run.
+* `Start Time`, required: the starting point for the query timeframe. This can be either a literal timestamp ("2023-01-01T00:00:00Z07:00") or a [Go-style duration string](https://pkg.go.dev/time#ParseDuration) such as "-10h" or "-20m".
+* `End Time`, required: the end point for the query timeframe. This can be either a literal timestamp ("2023-01-31T11:59:59Z07:00") or a [Go-style duration string](https://pkg.go.dev/time#ParseDuration) such as "0h" or "-5h".
```
Comment (ashnwade) [inline flows/nodes/runqueryadvanced.md:9]: Really minor observation:
Start Time has examples for hour and minute durations. The End Time uses hour for both examples. Recommend "-5m" instead.

File: flows/nodes/runqueryadvanced.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+Obviously it is rarely useful to run a flow over the exact same timeframe again and again, so users will rarely enter a literal timestamp by hand. However, reading start/end timestamps from *variables* means you can programmatically decide (using other nodes) a specific timeframe to search over.
+
+## Output
+
+The node inserts an object (named `search` by default) into the payload containing information about the search. The structure is identical to that output by the [Run a Query](runquery) node.
+
+## Example
+
+This node behaves exactly like the [Run a Query](runquery) node, except for the difference in timeframe specification. This section will therefore show a few examples of the timeframe could be configured:
```
Comment (ashnwade) [inline flows/nodes/runqueryadvanced.md:26]: missing word:
examples of **_how_** the timeframe could be configured

File: flows/nodes/runqueryadvanced.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+| Start | End | Explanation |
+|-------|-----|-------------|
+| 2023-01-01T00:00:00Z07:00 | 2023-01-31T11:59:59Z07:00 | Search over the entire month of January 2023 |
+| -24h | 0h | Search over the last day (equivalent to using the normal Run Query node with a duration of 24 hours) |
```
Comment (ashnwade) [inline flows/nodes/runqueryadvanced.md:31]: minor:
Run **_a_** Query

**Review by ashnwade — COMMENTED** [review]
Only very minor comments on the current PR, but I did check it out and build it to double check links. This is unrelated to the changes that you have made here, but changelog/5.0.0.md has a broken link for -ignore-glob. It should go to http://127.0.0.1:8000/ingesters/file_follow.html#ignore-glob
But we have to use the workaround for anchor links as documented on [this PR](https://github.com/gravwell/wiki/pull/555#issuecomment-1363264997). Even though it isn't related, I would recommend making the correction.

**Comment by john-floren-gravwell** [comment]
ptal

**Review by ashnwade — APPROVED** [review]
LGTM

**Pull request merged**